doc/tutorial/argument_parser/index.md

```diff
@@ -456,3 +456,87 @@ Add a seqan3::regex_validator to the first positional option that expects the `f
 \solution
 \snippet doc/tutorial/argument_parser/solution6.cpp file_validator
 \endsolution
+
+
+# The Version Update Feature
```

Comment:
I would call it "Update Notifications" since we are not really offering automatic updates.